src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+    private int number;
+
+    public String getColor() {
+        return color;
+    }
+
+    public void setColor(String color) {
+        this.color = color;
+    }
+
+    public int getNumber() {
+        return number;
+    }
+
+    public void setNumber(int number) {
+        this.number = number;
+    }
+
+    @Override
+    public String toString() {
+        return super.toString();
```

Comment:
there is no point in overriding toString() method and returning super.toString(), let's print actual info about our ball